xml/decoders/piko/PanePikoLichtDecoder_v5.1.xml

```diff
@@ -1345,6 +1367,21 @@
 	<group include="PIKO46551-18,PIKO_BR93_SUSI_Decoder">
 		<!-- Piko SmartDecoder XP5.1 -->
 		<!-- BR93 TT -->
+		<row>
+			<column>
+				<display item="Decoder_Test" format="checkbox"/>
+				<label>
+					<text>
+						<!-empty line-->
```

Comment:
This line
`<!-empty line-->`
should be
`<!--empty line-->`

A `-` is missing before the word `empty`.

Reply:
is done